test/spec.yml

```diff
@@ -1215,3 +1215,35 @@
     -   in: bn create node8 alreadyExists
         exit: 1
         err: "/home/dockeruser/test/binaris.yml already exists"
+
+- test: create configuration options (good-path)
+  steps:
+    -  in: bn create node8 testConfOpts --config.executionModel=exclusive --config.env.PATH --config.env.FOO=bar
+    -  in: |-
+           cat <<EOF >>/home/dockeruser/test/expected.yml
+           functions:
+             testConfOpts:
+               file: function.js
+               entrypoint: handler
+               executionModel: exclusive
+               runtime: node8
+               env:
+                 PATH: null
+                 FOO: bar
+           EOF
+    -  in: diff binaris.yml expected.yml
```

Comment:
I don't think this actually tests anything (there are no output verification blocks).

Reply:
it does. if it's non zero, the test fails. (as observed it during testing)

Reply:
Hmm, ok. My only comment is that this form of spec test is harder to discern at first glance (not specific to this PR but more of a general thought).